docker/scripts/New-AzureStackTestResources.ps1

```diff
@@ -0,0 +1,127 @@
+# Run Login-Environment.ps1 before running this script.
+
+[CmdletBinding()]
+param (
+    [string]
+    $Location = $ENV:Location
+)
+
+$resourceGroup = "azurestack-container-rg"
+$offer = "azurestack-container-offer"
+$plan = "azurestack-container-plan"
+
+function New-Quotas
+{
+    $serviceQuotas  = @()
+
+    $quotaName                  = "azstorquota"
+    $capacityInGb               = 1000
+    $numberOfStorageAccounts    = 200
```

Comment:
for simple tests you can revise the quotas to be just the bare minimum

Reply:
Currently there is no documentation for quota limits for azs admin modules, e.g., so I will leave the quotas as is for the test: https://docs.microsoft.com/en-us/powershell/module/azs.storage.admin/New-AzsStorageQuota?view=azurestackps-2.0.0